env.sample

```diff
@@ -17,6 +17,9 @@ DISCOVERY_USERNAME=03c25743-4728-448e-b3ed-3b198e6edd65
 DISCOVERY_PASSWORD=4LDhpX27q7QZ
```

Comment:
no, I changed all the IDs to be UUIDs (ran python -c "import uuid; print uuid.uuid4()" -- for the passwords I just changed a few characters in each.